modules/luci-mod-admin-full/luasrc/model/cbi/admin_system/leds.lua

```diff
@@ -121,13 +121,7 @@ usbport.size = 1
 function usbport.valuelist(self, section)
 	local port, ports = nil, {}
 	for port in util.imatch(m.uci:get("system", section, "port")) do
-		local b, n = port:match("^usb(%d+)-port(%d+)$")
-		if not (b and n) then
-			b, n = port:match("^(%d+)-(%d+)$")
-		end
-		if b and n then
-			ports[#ports+1] = "usb%u-port%u" %{ tonumber(b), tonumber(n) }
-		end
+			ports[#ports+1] = port
```

Comment:
Please fix the indentation level here (remove one tab)

Reply:
fixed